lib/pqos.h

```diff
@@ -1017,7 +1022,8 @@ PQOS_DEPRECATED
 #endif
 int pqos_alloc_reset(const enum pqos_cdp_config l3_cdp_cfg,
                      const enum pqos_cdp_config l2_cdp_cfg,
-                     const enum pqos_mba_config mba_cfg);
+                     const enum pqos_mba_config mba_cfg,
+                     const enum pqos_mba_config smba_cfg);
```

Comment:
The pqos_alloc_reset is deprecated. Please remove the changes in deprecated functions if any.

Reply:
Sure. Will do.